src/api/app/controllers/webui/requests/change_devel_projects_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Webui
+  module Requests
+    class ChangeDevelProjectsController < Requests::MainController
+      before_action :require_login
+      before_action :lockout_spiders
```

Comment:
Following @hennevogel's comment in one of my other PRs, I've realized that this is bad. I was also doing it. Blocking spiders instead of redirecting them to a page (the login page in this case) is hurting our SEO ranking. This line should be removed.

Reply:
Removed.